modules/github_integration/config/locales/en.yml

```
@@ -27,6 +27,20 @@
 #++
 
 en:
+  attributes:
+    additions_count: "Number of additions"
+    deletions_count: "Number of deletions"
+    comments_count: "Comment count"
+    github_html_url: "Pull Request URL"
+    github_app_owner_avatar_url: "Owner avatar URL"
+    review_comments_count: "Review comment count"
```

Shouldn't it be plural?

```suggestion
    comments_count: "Comments count"
    github_html_url: "Pull Request URL"
    github_app_owner_avatar_url: "Owner avatar URL"
    review_comments_count: "Review comments count"
```